docs/examples/pytorch/single_stage_detector/pytorch_ssd.rst

```diff
@@ -38,16 +38,14 @@ Usage
 
 .. code-block:: bash
 
-  python train.py [-h] [--data DATA] [--epochs EPOCHS] [--batch-size BATCH_SIZE]
-                  [--seed SEED] [--threshold THRESHOLD] [--iteration ITERATION]
-                  [--checkpoint CHECKPOINT] [--no-save]
-                  [--evaluation [EVALUATION [EVALUATION ...]]]
-
-For example, if you have COCO data in ``/data/coco2017`` and wish to train for 80 epochs you could use:
-
-  .. code-block:: bash
-
-    python train.py --data=/data/coco2017 --epochs=80
+  usage: main.py [-h] --data DATA [--epochs EPOCHS] [--batch-size BATCH_SIZE]
+                [--eval-batch-size EVAL_BATCH_SIZE] [--seed SEED]
+                [--evaluation [EVALUATION [EVALUATION ...]]]
+                [--multistep [MULTISTEP [MULTISTEP ...]]]
+                [--learning-rate LEARNING_RATE] [--momentum MOMENTUM]
+                [--weight-decay WEIGHT_DECAY] [--warmup WARMUP]
+                [--num-workers NUM_WORKERS] [--fp16] [--local_rank LOCAL_RANK]
+                [--data_pipeline {dali,no_dali}]
 
 All arguments with descriptions you can find in table below:
```

Comment:
Is this just an output of 
./main.py --help?

Reply:
More or less. The same thing we did for ResNet50 example, so I wanted it to be similar